src/app/shared/entity.service.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from 'ng-login';
+import { Entity } from 'ngx-login-client';
+export class EntityService {
```

Comment:
We probably need to revisit this part of the model, it's exceedingly painful to use, and I'm not sure this is the right approach (probably better to have // model hierachies). I will try to work on this during the week.